app/controllers/api/v1/registrations_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class RegistrationsController < ApplicationController
```

Comment:
@hoangmirs How about we keep using devise registrations controller for it?

Reply:
@ankitkalia1195 I planned to keep using Devise, but we only need to create the account and custom the response. That why I changed to not use it.
And the same case for forgot password